ansible/roles/automation/nightly_pr/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
            --repo_path {{ basedir }}/freeipa
            --flow {{ item.flow }}
            --branch {{ item.branch }}
-           --prci_config {{ prci_config }}
+           --prci_config ipatests/prci_definitions/{{ item.prci_config }}
```

Comment:
`ipatests/prci_definitions` can be replaced with `{{ prci_def_dir }}`.

Reply:
much better like that, thanks.